webgpu.yml

```diff
@@ -3184,8 +3190,8 @@ objects:
           TODO
         returns:
           doc: |
-            TODO
-          type: bool
+            Returns @ref WGPUStatus_Error (and leaves `limits` unchanged) if there was a @ref StructChainingError.
```

Comment:
What about if the device/adapter itself is an error? IMHO the error cases should be defined in the algorithm steps of the articles, and not call out a particular kind of error.

Reply:
This is meant to mirror `adapter.limits`/`device.limits` in JS, where there is no possibility of the JS-wrapper object being an error.

I'm OK with moving this stuff into articles if we want to write those, but it's easier to start documenting things here and move them to articles if they get larger.